docs/landing-page.md

```diff
@@ -0,0 +1,70 @@
+# Landing page
+
+_This feature is an experimental feature. The setup and configuration of the landing page needs manual code editing. There's no user interface in Admin panel to configure landing page._
+
+This guide will help you to setup and configure landing page. When the landing page is enabled, it will be served to your users from the root URL of your marketplace. The old root search view will be found from `/s` path.
+
+## Enabling landing page
+
+To enable the landing page, change the `clp_static_enabled` to `true`
+
+```yaml
+# config.yml
+
+clp_static_enabled: true
+```
+
+Restart the server and go to the marketplace root URL. You should now see the landing page with the example content.
+
+## Modifying the landing page content
+
+## Advanced configuration
+
+### Caching
+
+The landing page is cache heavily in order to make it lightning fast. There are two configurations that control caching:
+
+- `clp_cache_time`: The time in seconds how often the cache is invalidated. Default: 900 seconds, i.e. 15 minutes
+- `clp_static_released_version`: The current landing page version. The version number acts as a cache buster. The cache is in any case invalidated after `clp_cache_time`, but if you want to invalidate the cache sooner, increase the version number.
+
+### Images
+
+You can use your own images in several section in the landing page, e.g. hero section background, info section background and category background.
+
+`clp_asset_url` configuration controls where the assets are loaded from.
+
+#### Hosting the assets in /public (Recommended)
+
+#### Hosting the assets in S3 ()
+
+### Fonts
+
+Landing page is designed to be used with Proxima Nova Soft font, which is not freely available. If you want to use Proxima Nova Soft font in the landing page, you need to buy the font.
+
+`clp_font_proximanovasoft_url` configuration controls the path where the font is loaded from.
```

Comment:
Here's a reference to the font variable name which I commented about, so a reminder to change this one too.